container_files/Containerfile.services

```diff
@@ -99,7 +99,6 @@ WORKDIR /trustification/spog/ui
 
 RUN true \
     && npm ci \
-    && rustup target add wasm32-unknown-unknown \
     && trunk build --release --dist /public
```

Comment:
I got the logic for the changes on the other files, but is this change needed|intentional? just double-checking we are not pushing changes unintentionally (and I can not test our .tekton files other than just verifying the syntax of the file)

Reply:
Yea, it was a cleanup I found while working on this. It is a duplicate command in the container. Same as the Rust version. It's installing a Rust version of 1.74.0, but then that gets overridden by the toolchain file.

I should have called that out explicitly in the description.